src/features/SendAssets/__tests__/SignTransactionLedger.test.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { simpleRender, waitFor, act } from 'test-utils';
+
+import { fTxConfig, fNetwork } from '@fixtures';
+import { WalletId } from '@types';
+import { translateRaw } from '@translations';
+import { WALLETS_CONFIG } from '@config';
+
+import SignTransaction from '../components/SignTransaction';
+
+const defaultProps: React.ComponentProps<typeof SignTransaction> = {
+  txConfig: {
+    ...fTxConfig,
+    senderAccount: { ...fTxConfig.senderAccount, wallet: WalletId.LEDGER_NANO_S }
+  },
+  onComplete: jest.fn()
+};
+
+const getComponent = () => {
+  return simpleRender(<SignTransaction {...defaultProps} />);
+};
+
+const getHeader = (wallet: WalletId) => {
+  return translateRaw('SIGN_TX_TITLE', {
+    $walletName: WALLETS_CONFIG[wallet].name
+  });
+};
+
+const mockGetAddress = jest.fn().mockImplementation(() => ({
+  publicKey: defaultProps.txConfig.senderAccount.address,
+  chainCode: fNetwork.chainId
+}));
+const mockSign = jest.fn().mockImplementation(() => Promise.resolve({ v: 10, r: 2, s: 4 }));
+jest.mock('@ledgerhq/hw-transport-u2f');
+jest.mock('@ledgerhq/hw-app-eth', () => {
+  return jest.fn().mockImplementation(() => ({
+    getAddress: mockGetAddress,
+    signTransaction: mockSign
+  }));
+});
+
+describe('SignTransaction', () => {
+  beforeEach(() => {
+    jest.useFakeTimers();
+  });
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  test('Can handle Ledger signing', async () => {
+    const { getByText } = getComponent();
+    const selector = getHeader(WalletId.LEDGER_NANO_S);
+    expect(getByText(selector)).toBeInTheDocument();
+    act(() => {
+      jest.advanceTimersByTime(3001);
+    });
+    await waitFor(() =>
+      expect(defaultProps.onComplete).toBeCalledWith(
+        new Buffer([
```

Comment:
Extensive comment explaining what this corresponds to.
If useful to reuse move to __fixtures__

Reply:
I don't think it's useful to reuse, it's very much dependant on the signing payload.